src/app/docs/concepts/router/page.mdx

```diff
@@ -0,0 +1,48 @@
+
+## Router
+
+To make composing protocols easier, iroh includes a _router_ for composing together multiple protocols. {{className: 'lead'}}
+
+The router implements the [accept loop](/docs/concepts/protocol#the-accept-loop) on your behalf, and routes incoming connections to the correct protocol based on the ALPN. We recommend using the router to compose protocols, as it makes it easier to add new protocols to your application.
+
+We use the term _router_ because it mimics what an HTTP server would do with a URL-based router: It routes incoming connections to the correct protocol based on the ALPN.
```

Comment:
```suggestion
We use the term _router_ because it mimics what an HTTP server would do with an URL-based router: It routes incoming connections to the correct protocol based on the ALPN.
```

Debatable. But I think this is nicer, as I think most people say "URL" as "uh-rl" or "you-are-el".